sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java

```diff
@@ -196,6 +196,46 @@ public CosmosContainerProperties setConflictResolutionPolicy(ConflictResolutionP
         return this;
     }
 
+    /**
+     * Gets the collection's default time-to-live value.
+     *
+     * @return the default time-to-live value in seconds.
+     */
+    public Integer getDefaultTimeToLive() {
```

Comment:
shouldn't we include the unit?

`getDefaultTimeToLiveInSeconds()` ?

Reply:
We can if we want to, I am fine with it. 
Though the API has been called `getDefaultTimeToLive()` in both v2 and v3. 
Should we change it ?

Reply:
Let's decide what is the right thing to do and we do that for v4 as v4 is not released yet.

Reply:
Agreed, @kirankumarkolli @mbhaskar @David-Noble-at-work  do you guys have any opinion on this ?
Mo and I are in agreement of changing it to `getDefaultTimeToLiveInSeconds()` and `setDefaultTimeToLiveInSeconds()`

Reply:
Done.